test/ConcatSource.js

```diff
@@ -62,4 +62,45 @@ describe("ConcatSource", function() {
 			map: expectedMap2
 		});
 	});
+
+	it('should be able to handle strings for all methods', function() {
+		var source = new ConcatSource(
+			new RawSource("Hello World\n"),
+			new OriginalSource("console.log('test');\nconsole.log('test2');\n", "console.js")
+		);
```

Comment:
You didn't pass strings to the ConcatSource. Does this test test the added behavior?